### ISSUE TITLE
Subdue default times do not provide enough sub-second resolution

### ISSUE THREAD

**Issue by jtschelling** [sensu/sensu#1964]
When subdue end occurs before begin the default values that sensu provides doesn’t extend to milliseconds which is causing issues when a day rolls over to the next. 

## Expected Behavior
Given the a check's subdue attributes as following:
```
"days": {
  "wednesday": {
    [
      "begin": "22:30 UTC",
      "end": "11:55 UTC"
    ]
  }
}
```

A check that executes at time `23:59:59.999` should be subdued.

## Current Behavior
Given the above check attributes, a check that executes at time `23:59:59.999 UTC` will not be subdued.

## Possible Solution
Putting up a PR

## Steps to Reproduce (for bugs)
This test in `./spec/utilities_spec.rb` should not error

```
    Timecop.freeze(Time.parse('23:59:59.999 UTC'))
    check[:subdue][:days][:all] = [
      {
        :begin => '2019-10-17 22:30:00 +0000',
        :end => "2019-10-17 11:55:00 +0000"
      }
    ]
    expect(check_subdued?(check)).to be(true)
```
Part of our problem is the number of checks we have running, the more checks we run the higher the probability of this problem biting us.

## Context
We have 100s of checks using subdue attributes running on 30s intervals so the probability is very high that some checks will run between the times `23:59:59 UTC` and the next day's `00:00:00 UTC`, these have been alerting when that should not occur.

## Your Environment
Sensu-core 1.8

**Comment by jtschelling** [comment]
This was fixed in sensu-go https://github.com/sensu/sensu-go/blob/a786ceeb603601f89a39276824de346b098ad5e4/api/core/v2/time_window.go#L81-L95

**Comment by portertech** [comment]
Merged #1965
